libs/backend/src/cast_vote_records/export.ts

```diff
@@ -423,6 +425,76 @@ async function exportSignatureFileToUsbDrive(
   return ok();
 }
 
+/**
+ * Updates the creation timestamp of a directory and its children files (assuming no
+ * sub-directories) using the one method guaranteed to work:
+ * ```
+ * cp -r <directory-path> <directory-path>-temp
+ * rm <directory-path>
+ * mv <directory-path>-temp <directory-path>
+ * ```
+ */
+async function updateCreationTimestampOfDirectoryAndChildrenFiles(
+  directoryPath: string
+): Promise<void> {
+  const tempDirectoryPath = `${directoryPath}-temp`;
+  await fs.mkdir(tempDirectoryPath);
+  const fileNames = (await fs.readdir(directoryPath, { withFileTypes: true }))
+    .filter((entry) => entry.isFile())
+    .map((entry) => entry.name);
+  for (const fileName of fileNames) {
+    await fs.copyFile(
+      path.join(directoryPath, fileName),
+      path.join(tempDirectoryPath, fileName)
+    );
+  }
+  await fs.rm(directoryPath, { recursive: true });
+  await fs.rename(tempDirectoryPath, directoryPath);
+}
+
+/**
+ * File creation timestamps could reveal the order in which ballots were cast. To maintain
+ * voter privacy, every time a ballot is cast, we randomly select 1 or 2 existing cast vote records
+ * and update their creation timestamps to the present. This approach is equivalent to taking
+ * random ballots from a stack and placing them on top every time a new ballot is added to the
+ * stack, a kind of shuffling as we go.
+ *
+ * This shuffling as we go is irrelevant for full exports, where we already iterate over cast vote
+ * records in an order independent of creation timestamp.
+ */
+async function maintainVoterPrivacy(
+  exportContext: ExportContext
+): Promise<void> {
```

Comment:
who needs election guard when you can have a javascript function called `maintainVoterPrivacy`